circle.yml

```diff
@@ -7,6 +7,7 @@ machine:
       6.9.1
 dependencies:
   override:
+    - python buildscript.py
```

Comment:
No extra need to call this here once we call `bin/bundle-extensions` from package.json.